fontforgeexe/charview.c

```diff
@@ -488,12 +488,13 @@ const int input_em_cnt = sizeof(input_em)/sizeof(input_em[0])-1;
 
 void CVDrawRubberRect(GWindow pixmap, CharView *cv) {
     GRect r;
+    int tabnumber = GTabSetGetSel(cv->tabs);
```

Comment:
This is fine, but one way to make this less verbose is to do something like

```c
CharViewTab* tab = &cv->cvtabs[GTabSetgetSel(cv->tabs)]; // I didn't check what the type of cvtabs  is
...
tab->xoff
...
```

Reply:
I would say, &ldquo;do I have to?&rdquo; Most of the substitutions were done automatically. It's the same to the computer, and people can see the git blame to see why it's that way.

Reply:
Yes, I'm fine with how it's currently done, just stating my preference.

Reply:
Thinking about this a bit more, and in consideration of that bitmap view issue, this actually may not be the best solution. Will need to look a bit further to think of what the alternatives are.